app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -14,21 +14,25 @@ def show
   # GET /wrangling_guidelines/new
   def new
     @wrangling_guideline = WranglingGuideline.new
+    authorize @wrangling_guideline
```

Comment:
Anywhere we're doing `@wrangling_guideline = WranglingGuideline.new` or `@wrangling_guideline = WranglingGuideline.find`, we should be able to just stick `authorize` in front of `WranglingGuideline` to make things a bit neater.